lib/kadm5/prune_c.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * TODO licence
+ */
+
+#include "kadm5_locl.h"
+
+RCSID("$Id$");
+
+kadm5_ret_t
+kadm5_c_prune_principal(void *server_handle, krb5_principal princ, int prunekvno)
+{
+    kadm5_client_context *context = server_handle;
+    kadm5_ret_t ret;
+    krb5_storage *sp;
+    unsigned char buf[1024];
+    int32_t tmp;
+    krb5_data reply;
+
+    ret = _kadm5_connect(server_handle);
+    if(ret)
+        return ret;
+
+    sp = krb5_storage_from_mem(buf, sizeof(buf));
+    if (sp == NULL) {
+        krb5_clear_error_message(context->context);
+        return ENOMEM;
+    }
+    krb5_store_int32(sp, kadm_prune);
```

Comment:
Error checking is needed.

Reply:
code is copied from other features. i'm willing to work it out.

Reply:
Thanks for pointing that out.  I'll fix the others, but do please fix this one.

Reply:
remarked as unresolved to keep track of the things i've promissed/asked to fix

Reply:
@nicowilliams please review commit 0d655f5 and resolve